### PR TITLE
Updating to latest template

### PR DESCRIPTION
**Pull request by paulomarg**
Closes https://github.com/Shopify/first-party-library-planning/issues/409

This updates the result of the "getting started" tutorial example app to start from the updated template that uses `@shopify/shopify-app-express`.

### CODE REVIEW

File: qr-code/node/web/frontend/pages/index.jsx [hunk 1]
```diff
@@ -6,8 +6,8 @@ import {
   Page,
   SkeletonBodyText,
 } from "@shopify/polaris";
-import { useAppQuery } from "../hooks";
 import { QRCodeIndex } from "../components";
+import { useAppQuery } from "../hooks";
```
Comment (shainaraskas) [inline qr-code/node/web/frontend/pages/index.jsx:10]: looks like this might be missing here: https://shopify-dev.shopify-dev-minimal-72uq.paulo-margarido.us.spin.dev/apps/getting-started/build-app-example/frontend#create-your-home-page-with-an-empty-state

File: qr-code/node/web/frontend/pages/index.jsx [hunk 2]
```diff
@@ -23,11 +23,11 @@ export default function HomePage() {
...
-      react-query provides stale-while-revalidate caching.
-      By passing isRefetching to Index Tables we can show stale data and a loading state.
-      Once the query refetches, IndexTable updates and the loading state is removed.
-      This ensures a performant UX.
-    */
+    react-query provides stale-while-revalidate caching.
+    By passing isRefetching to Index Tables we can show stale data and a loading state.
+    Once the query refetches, IndexTable updates and the loading state is removed.
+    This ensures a performant UX.
+  */
```
Comment (shainaraskas) [inline qr-code/node/web/frontend/pages/index.jsx:30]: these comment fences should be aligned - I think you can just revert this change

File: qr-code/node/web/helpers/qr-codes.js [hunk 3]
```diff
@@ -58,19 +52,18 @@ export async function getQrCodeOr404(req, res, checkDomain = true) {
...
-  discountCode: string
-  destination: string
+Expect body to contain
+title: string
+productId: string
+variantId: string
+handle: string
+discountId: string
+discountCode: string
+destination: string
```
Comment (shainaraskas) [inline qr-code/node/web/helpers/qr-codes.js:66]: should indent the comment body for consistency with the other comments

File: qr-code/node/web/qr-codes-db.js [hunk 4]
```diff
@@ -72,17 +61,6 @@ export const QRCodesDB = {
   ) {
     await this.ready;
 
-    this.__validateQRCode({
```
[the remark this answers is not in the document]
Comment (shainaraskas) [reply qr-code/node/web/qr-codes-db.js:75]: no strong opinion

**Review by shainaraskas — APPROVED** [review]
no blocking issues

File: qr-code/node/web/qr-codes-db.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (paulomarg) [reply qr-code/node/web/qr-codes-db.js:75]: Since we never had this in the template, let's just keep it removed from here as it makes the code simpler to understand.

File: qr-code/node/web/frontend/pages/index.jsx (hunk 1 above)
Comment (paulomarg) [reply qr-code/node/web/frontend/pages/index.jsx:10]: This particular line gets added later on: https://shopify-dev.shopify-dev-minimal-72uq.paulo-margarido.us.spin.dev/apps/getting-started/build-app-example/connect#step-4-connect-the-list-view-to-display-saved-qr-codes
